app/src/main/java/pl/wp/quiz/adapter/QuizDetailsAdapter.java

```diff
@@ -23,12 +31,26 @@
             quizInfo = itemView.findViewById(R.id.quiz_info);
             quizImage = itemView.findViewById(R.id.quiz_image);
         }
+
+        @Override
+        public void onImageLoaded(String source) {
+            Log.d(TAG, "onImageLoaded: source load");
+//            quizImage.setImageDrawable(sourceToDrawable(source));
+        }
+
+        private Drawable sourceToDrawable(String source) {

```

Comment:
this method always returns null. Add TODO if this will be implemented later or remove it